src/runtime_src/core/tools/common/ReportAieShim.cpp

```diff
@@ -29,6 +31,7 @@
 
 
 namespace qr = xrt_core::query;
+std::vector<std::string> aieTileList;

```

Comment:
Are we adding a global variable here?

Reply:
Not required to be a global variable. Moved it as a local variable .. Thanks